File: Backend/src/Trackable.Services/DataClasses/DispatchingResults.cs
using System.Collections.Generic;
using Trackable.Models;

namespace Trackable.Services
{
    public class DispatchingResults
    {
        public IEnumerable<string> ItineraryText { get; set; }

        public IEnumerable<Point> ItineraryPoints { get; set; }

        public IEnumerable<string> ItineraryDistance { get; set; }

        public IEnumerable<Point> RoutePoints { get; set; }

        public IEnumerable<Point> AlternativeCarRoutePoints { get; set; }
    }
}

File: Frontend/src/app/dispatching/dispatching-results.ts
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

import { Point } from '../shared/point';

export class DispatchingResults {
    itineraryText: string[];
    itineraryPoints: Point[];
    itineraryDistance: string[];
    routePoints: Point[];
    alternativeCarRoutePoints: DispatchingResults[];
}
